packages/vaex-core/vaex/expression.py

```diff
@@ -462,6 +462,13 @@ def reduce(a, b):
     def unique(self):
         return self.ds.unique(self.expression)
 
+    def nunique(self, dropna=True):
+        """Return the number of unique elements in the expression."""
+        n = len(self.unique())
+        if dropna:
+            return n - 1
```

Comment:
this won't always work, what if there is no missing values or nans? Also, I think we should have a consistent argument set each time: `dropna=True, dropnan=True, dropmissing=True` maybe? @xdssio: what do you think?

Reply:
Ah yes... also the presence of `None` values also messes the counts up.. but I just noticed that `None` values are not dropped by `dropna`.

If we want to be similar to `pandas`, we should adopt `dropna` as Not Available (n/a)